intellij-idea-code-style.xml

```diff
@@ -1,41 +1,41 @@
-<code_scheme name="PatternPediaAPI">
-    <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="999" />
-    <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="999" />
+<code_scheme name="PatternAtlasAPI">
+    <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="999"/>
+    <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="999"/>
     <option name="PACKAGES_TO_USE_IMPORT_ON_DEMAND">
-        <value />
+        <value/>
     </option>
     <option name="IMPORT_LAYOUT_TABLE">
         <value>
-            <package name="java" withSubpackages="true" static="false" />
-            <emptyLine />
-            <package name="com.patternpedia.api" withSubpackages="true" static="false" />
-            <emptyLine />
-            <package name="" withSubpackages="true" static="false" />
-            <emptyLine />
-            <package name="" withSubpackages="true" static="true" />
+            <package name="java" withSubpackages="true" static="false"/>
+            <emptyLine/>
+            <package name="io.github.ust.quantil.patternatlas.api" withSubpackages="true" static="false"/>
```

Comment:
Wouldn't this be `io.github.patternatlas.api`? (as https://patternatlas.github.io/pattern-atlas-api/)

Reply:
Somehow this didn't get updated...